src/roles.js

```diff
@@ -269,10 +270,8 @@ export class Roles extends TypedEmitter {
    * @returns {Promise<Role>}
    */
   async getRole(deviceId) {
-    const roleAssignment = await this.#dataType.getByDocId(deviceId, {
-      mustBeFound: false,
-    })
-    if (!roleAssignment) {
+    const roleRecord = await getByDocIdIfExists(this.#dataType, deviceId)
```

Comment:
Not strictly related, but I renamed this to make the diff for #188 a little smaller.